src/components/profileEdit/ProfileInfo.tsx

```
@@ -142,13 +200,37 @@ export default function ProfileInfo() {
+        initPasswords();
+        setSuccessEditPassword(false)
+    }, [editingPassword])
+
+    useEffect(() => {
+        setNewName(profile.name)
+        setProfileImage(profile.imageUrl)
+    }, [profile])
+
+    useEffect(() => {
+        setNewIntroduction(intro.intro)
+        setNewUrls(intro.urls)
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)